### PR TITLE
III-3654 Update the status of an offer

### PR DESCRIPTION
**Pull request by LucWollants**
### Added
- Update the status of an offer

---

Ticket: https://jira.uitdatabank.be/browse/III-3654

---
### Todo
- [x] Add tests for the new scenarios

### CODE REVIEW

File: src/Offer/Commands/Status/AbstractUpdateStatus.php [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    }
+
+    public function getItemId(): string
+    {
+        return $this->eventId;
+    }
+
+    public function getPermission(): Permission
+    {
+        return  Permission::AANBOD_MODEREREN();
```
Comment (LucWollants) [inline src/Offer/Commands/Status/AbstractUpdateStatus.php:41]: Could it be that this requires `AANBOD_BEWERKEN` permissions?

File: tests/Place/CommandHandlers/UpdateStatusHandlerTest.php [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+            ->withAggregateId($id)
+            ->given([$this->getPlaceCreated($id, $initialCalendar)])
+            ->when($command)
+            ->then([$expectedEvent]);
+    }
+
+    /**
+     * @test
+     */
+    public function it_will_update_status_of_sub_events(): void
```
Comment (jdrieghe) [inline tests/Place/CommandHandlers/UpdateStatusHandlerTest.php:72]: I'm not sure how relevant this scenario is for `Place`?

Comment (bertramakers) [reply tests/Place/CommandHandlers/UpdateStatusHandlerTest.php:72]: Not relevant indeed, places can only be `permanent` or `periodic`

**Comment by bertramakers** [comment]
Only suggestion I have is https://github.com/cultuurnet/udb3-silex/pull/537#discussion_r551280536

**Review by jdrieghe — APPROVED** [review]

**Pull request merged**